files/default/audit_report.rb

```diff
@@ -98,6 +116,22 @@ def gather_nodeinfo
         }
       end
 
+      # this is a helper methods to extract the profiles we scan and hand this
+      # over to the reporter in addition to the `json-min` report. `json-min`
+      # reports do not include information about the source of the profiles
+      # TODO: should be available in inspec `json-min` reports out-of-the-box
+      # TODO: raise warning when not a compliance-known profile
+      def cc_profile_index(profiles)
+        cc_profiles = tests_for_runner(profiles).map { |profile| profile[:compliance] if profile[:compliance] }.uniq.compact
```

Comment:
In this case we would need:

`tests_for_runner(profiles).select { |profile| profile[:compliance] }.map { |profile| profile[:compliance].uniq.compact`

I think it reads better, although we loop more often. Thanks @srenatus